node-require/calculate.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+process.argv.add = add(process.argv[2], process.argv[4]);
+process.argv.subtract = subtract(process.argv[2], process.argv[4]);
+process.argv.multiply = multiply(process.argv[2], process.argv[4]);
+process.argv.divide = divide(process.argv[2], process.argv[4]);
```

Comment:
In general we don't want to modify the `argv` property of `process`. Also, try to avoid computing values until you actually need them.